.gitignore

```
@@ -133,3 +133,7 @@ dmypy.json
 
 # Auto scanner file
 .autoscanner-state-file
+
+signal_scanner_bot/.diff-autoscan-state-file
```

Maybe we could take a similar approach with the env files? E.g. this could be `.diff.autoscanner-state-file` and it would be ignored:
```suggestion
*.autoscanner-state-file
```

---

Ah, this probably should be cleaned up actually. The .diff-autoscanner-state-file looks like it was just a test to make sure that the input was working as expected, and it got rolled into a change at some point, even though it shouldn't have been. I think for the sake of simplicity we should just leave this as `.autoscanner-state-file` because that's the default, and there's no point trying to catch every possible input someone could put there.